tests/operator_test.go

```diff
@@ -1249,8 +1249,9 @@ spec:
 				}, 90*time.Second, 1*time.Second).Should(BeTrue())
 			}
 
-			By("Verifying all migratable vmi workloads are updated via live migration")
-			verifyVMIsUpdated(migratableVMIs, launcherSha)
+			// QUARANTINED Logic - tracked by issue https://github.com/kubevirt/kubevirt/issues/5228
+			//By("Verifying all migratable vmi workloads are updated via live migration")
+			//verifyVMIsUpdated(migratableVMIs, launcherSha)
```

Comment:
You can keep this running (and avoid the comment-out code) using an XFail.

It will look like this:
```
assert.XFail("https://github.com/kubevirt/kubevirt/issues/5228", func() {
    By("Verifying all migratable vmi workloads are updated via live migration")
    verifyVMIsUpdated(migratableVMIs, launcherSha)
})
```